ui/src/hooks/useKubernetes.ts

```diff
@@ -393,10 +389,6 @@ export const handleTerminalMessage = (ws: WebSocket, event: MessageEvent, create
 
         break;
       }
-      case 'TERMINAL_ERROR': {
-        currentTerminal.write(info.err);
-        break;
-      }
       case 'TERMINAL_SHARE_JOIN': {
         const data = JSON.parse(info.data);
         const k8s_id: string = info.k8s_id;
```

Comment:
There do not appear to be any differences in code structure that would necessitate corrections. Please review the sourcecode with a more detailed understanding of what is being achieved there. If you identify specific sections where changes could potentially simplify implementation, provide a description and I will assist further. No optimization suggestions were provided as no issues have been spotted so far.